easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-2.2.2.6-gompi-2022b.eb

```diff
@@ -0,0 +1,57 @@
+# Updated to v2.1.3 by
+# R.QIAO <rob.qiao@flinders.edu.au>
+# DeepThought, Flinders University
+
+easyblock = 'CMakeMake'
+
+name = 'IQ-TREE'
+version = '2.2.2.6'
+
+# HTTPS is not working
+homepage = 'http://www.iqtree.org/'
+description = """Efficient phylogenomic software by maximum likelihood"""
+
+toolchain = {'name': 'gompi', 'version': '2022b'}
+# Including 'usempi' will take precedence and override IQTREE_FLAGS and produces only 'iqtree-mpi' binary
+
+source_urls = ['https://github.com/iqtree/iqtree2/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'IQ-TREE-2.1.2_use_EB_LSD2.patch',
+    'IQ-TREE-2.2.1_fix-mpi.patch',
+]
+checksums = [
+    {'v2.2.2.6.tar.gz': 'f5d10c033f2ed1c81bb473f572c7a9fe72a275a48e2e93ad74e5f1c3d32fa8b9'},
+    {'IQ-TREE-2.1.2_use_EB_LSD2.patch': 'daa2ab12d44e26eb5607c4ed6acb9d970e230a83dabcf21461f37bc48263b816'},
+    {'IQ-TREE-2.2.1_fix-mpi.patch': '9ead6808efd11d4c01dd265cca6094cffd6377746d3b2fc84b43d2faeee0777c'},
+]
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Eigen', '3.4.0'),
+]
+dependencies = [
+    ('zlib', '1.2.12'),
+    ('Boost', '1.81.0'),
+    ('LSD2', '2.4.1'),
+]
+
+local_conf_opts = ' -DUSE_LSD2=ON '
+configopts = [
+    '-DIQTREE_FLAGS=omp' + local_conf_opts,
+    '-DIQTREE_FLAGS=mpi -DCMAKE_C_COMPILER="$MPICC" -DCMAKE_CXX_COMPILER="$MPICXX"' + local_conf_opts,
+]
+
+sanity_check_paths = {
+    'files': ['bin/iqtree2', 'bin/iqtree2-mpi'],
+    'dirs': [],
+}
+
+sanity_check_commands = [
+    "iqtree2 --help",
+    "mkdir -p $TMPDIR/{test-omp,test-mpi}",
+    "cd $TMPDIR/test-omp && cp -a %(installdir)s/example.phy . && iqtree2 -s example.phy",
```

Comment:
maybe this needs a `-redo` as well, because the results of the `IQ-TREE-2.2.2.6-gompi-2022a.eb` sanity checks are still in the same location?
if that's the case probably best to just always add `-redo`?